worlds/tits_the_3rd/client/client.py

```diff
@@ -245,6 +267,11 @@ async def tits_the_3rd_watcher(ctx: TitsThe3rdContext):
             await ctx.wait_for_ap_connection()
             continue
 
```

Comment:
Recommend:

```
if not ctx.received_initial_server_data():
  await ctx.wait_for_initial_connection_info()
  
if not ctx.game_interface:
  ctx.init_game_interface()
```

And after which below we don't need any of the checks to know if we received the initial server info, as its always a requirement to make it to the main part of the loop.

You can steal wait_for_initial_connection_info from the RR client https://github.com/tdkollins/Archipelago-Rabi-Ribi/blob/b4ffc7180f6099c96411fe41fc7183466a517cbf/worlds/rabi_ribi/client/client.py#L219